zipkin-lens/src/components/DependenciesPage/DependenciesPage.test.js

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2015-2020 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+/* eslint-disable no-shadow */
+import React from 'react';
+import { createMemoryHistory } from 'history';
+import moment from 'moment';
+import { fireEvent, waitForElement } from '@testing-library/react';
+import fetchMock from 'fetch-mock';
+
+import render from '../../test/util/render-with-default-settings';
+import DependenciesPage from './DependenciesPage';
+
+jest.mock('@material-ui/pickers', () => {
+  // eslint-disable-next-line global-require
+  const moment = require('moment');
+  return {
+    // eslint-disable-next-line react/prop-types
+    KeyboardDateTimePicker: ({ value, onChange }) => (
+      <input
```

Comment:
Can you make sure all new files that use JSX end with `.jsx` extension? And to confirm I guess you're waiting to do typescript in another PR?